Client/Frontend/Home/FxPocketHomeHorizontalCell.swift

```diff
@@ -0,0 +1,186 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+class FxPocketHomeHorizontalCellViewModel {
+    var title: String { story.title }
+    var imageURL: URL { story.imageURL }
+    var url: URL? { story.url }
+    var sponsor: String? { story.sponsor }
+    var description: String {
+        if let sponsor = story.sponsor {
+            return sponsor
+        } else {
+            return "\(story.domain) • \(String.localizedStringWithFormat(String.FirefoxHomepage.Pocket.NumberOfMinutes, story.timeToRead ?? 0))"
+        }
+    }
+
+    var onTap: (IndexPath) -> Void = { _ in }
+
+    private let story: PocketStory
+
+    init(story: PocketStory) {
+        self.story = story
+    }
+}
+
+// MARK: - FxHomeHorizontalCell
+/// A cell used in FxHomeScreen's Jump Back In and Pocket sections
+class FxPocketHomeHorizontalCell: UICollectionViewCell, ReusableCell {
+
+    // MARK: - UI Elements
+    let heroImageView: UIImageView = .build { imageView in
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = FxHomeHorizontalCellUX.generalCornerRadius
+        imageView.backgroundColor = .clear
+    }
+
+    private let titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline,
+                                                                   maxSize: FxHomeHorizontalCellUX.titleFontSize)
+        label.numberOfLines = 2
+    }
+
+    private lazy var sponsoredStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [sponsoredIcon, sponsoredLabel])
+        stack.axis = .horizontal
+        stack.spacing = 8
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        return stack
+    }()
+
+    private lazy var sponsoredLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline,
+                                                                       maxSize: FxHomeHorizontalCellUX.sponsoredFontSize)
+        label.textColor = .secondaryLabel
+        label.numberOfLines = 1
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var sponsoredIcon: UIImageView = {
+        let image = UIImageView(image: UIImage(named: "sponsored-star"))
```

Comment:
Could the new image be added to ImageIdentifiers?

Reply:
Updated